### ISSUE TITLE
Is it possible to make the bot only repost pictures?

### ISSUE THREAD

**Issue by Reddyy98** [MysteryPancake/Discord-Reposter#7]
So as the title says i want the bot to ONLY repost pictures and ignore text messages, is that possible to do?

**Comment by MysteryPancake** [comment]
This can be done by removing [lines 231-233](https://github.com/MysteryPancake/Discord-Reposter/blob/master/reposter.js#L231-L233) and [lines 239-246](https://github.com/MysteryPancake/Discord-Reposter/blob/master/reposter.js#L239-L246). If you also want to remove the username, then remove [lines 222-230](https://github.com/MysteryPancake/Discord-Reposter/blob/master/reposter.js#L222-L230). I will probably add a toggle for this later

**Comment by MysteryPancake** [comment]
If you want to repost the image URL rather than downloading the image and reuploading it again, then change [line 236](https://github.com/MysteryPancake/Discord-Reposter/blob/master/reposter.js#L236) to `await send(webhook ? webhook : channel, attachment.url, message.reactions);`

**Comment by Reddyy98** [comment]
> 
> 
> This can be done by removing [lines 231-233](https://github.com/MysteryPancake/Discord-Reposter/blob/master/reposter.js#L231-L233) and [lines 239-246](https://github.com/MysteryPancake/Discord-Reposter/blob/master/reposter.js#L239-L246). If you also want to remove the username, then remove [lines 222-230](https://github.com/MysteryPancake/Discord-Reposter/blob/master/reposter.js#L222-L230). I will probably add a toggle for this later

thanks for the reply. okay so i removed lines 231-233 and 239-246 and yes it works, it doesn't repost text messages, but if i don't remove lines 222-230 to remove the username, the bot will repost only the username everytime a message is sent. i want to keep the username so i know who posted that precise picture so is there a fix to this?

**Comment by MysteryPancake** [comment]
Yes, sorry, I didn't think of that. I think it will work if you replace [the sendMessage function](https://github.com/MysteryPancake/Discord-Reposter/blob/master/reposter.js#L220-L247) with this instead:
````
async function sendMessage(message, channel, webhook, author) {
	if (inactive(channel.id, message.channel.id)) return;
	if (message.attachments.size) {
		if (message.author.id !== author) {
			if (webhook) {
				await webhook.edit(replaceAll(channel, niceName(channel, message.channel, message.author)), message.author.displayAvatarURL).catch(console.error);
			} else {
				await channel.send("**" + replaceAll(channel, niceName(channel, message.channel, message.author)) + "**").catch(console.error);
			}
		}
		for (const attachment of message.attachments.values()) {
			await send(webhook ? webhook : channel, attachment.filesize > 8000000 ? attachment.url : { files: [attachment.url] }, message.reactions);
		}
	}
}
```
This is just the username part of the code pasted into the bit checking the attachments. I will eventually get around to adding an option for this, but hopefully this will work for now.
